renovate.json

```diff
@@ -276,7 +277,7 @@
       "matchStrings": [
         "task_bundle: (?<depName>quay\\.io/konflux-ci/(?:konflux-vanguard|tekton-catalog)/[^:]+):(?<currentValue>[^@]*)@(?<currentDigest>sha256:[a-f0-9]{64})"
       ],
-      "autoReplaceStringTemplate": "task_bundle: {{depName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}",
+      "autoReplaceStringTemplate": "task_bundle: {{depName}}:{{currentValue}}@{{newDigest}}",
```

Comment:
Does this actually do what you want? Won't it just end up using the digest that corresponds to `newValue` while keeping the old, misleading tag?

Reply:
This is to not update the version but update the digest since the task_bundle in external folder don't need to upgrade the version, such as this 0.4 version [deprecated-image-check.yaml](https://github.com/konflux-ci/build-definitions/blob/main/external-task/deprecated-image-check/0.4/deprecated-image-check.yaml)